rero_ils/modules/patrons/jsonschemas/form_patrons/patron-v0.0.1.json

```diff
@@ -74,13 +74,23 @@
           {
             "key": "roles",
             "notitle": "true",
-            "description": "Is the user a patron or/and a librarian?",
+            "description": "User role(s) ?",
```

Comment:
`"description": "What is/are the role(s) of the user?"`

Reply:
done